src/main/java/org/mule/service/http/impl/service/server/grizzly/ResponseStreamingCompletionHandler.java

```diff
@@ -103,7 +107,15 @@ private int calculateBufferSize(FilterChainContext ctx, ClassLoader ctxClassLoad
   }
 
   public void start() throws IOException {
-    sendInputStreamChunk();
+    Thread thread = currentThread();
+    ClassLoader currentClassLoader = thread.getContextClassLoader();
+    ClassLoader newClassLoader = getEffectiveCtxClassLoader();
```

Comment:
Avoid doing these calls if `REPLACE_CONTEXT_CLASSLOADER` is already false.

Reply:
this happens in other places as well

Reply:
Done.
I updated the spreadsheet with the metrics in both cases.